services/QuillLMS/config/initializers/rails_admin.rb

```diff
@@ -142,3 +142,13 @@
     config.excluded_models.push(*MODELS_TO_EXCLUDE)
   end
 end
+
+# Monkey patch for a known issue: RailsAdmin tries to parse search strings as JSON
+# https://github.com/railsadminteam/rails_admin/issues/2502
```

Comment:
This seems to use half of the solution listed [here](https://github.com/railsadminteam/rails_admin/issues/2502#issuecomment-504612818). Is there a reason we aren't only using this piece?

Reply:
That part was not necessary during my QA testing, but I added it in, in case there's some nuance that the author caught.